integrationExamples/gpt/gdpr_indefinite_timeout_exampe.html

```diff
@@ -0,0 +1,155 @@
+                          commandQueue.push({
+                              command: command,
+                              parameter: parameter,
+                              callback: callback
+                          });
+                      }
+                  }
+                  cmp.commandQueue = commandQueue;
+                  cmp.receiveMessage = function(event) {
+                      var data = event && event.data && event.data.__cmpCall;
+                      if (data) {
+                          commandQueue.push({
+                              callId: data.callId,
+                              command: data.command,
+                              parameter: data.parameter,
+                              event: event
+                          });
+                      }
+                  };
+                  cmp.config = {
+                      //
+                      // Modify config values here
+                      //
+                      // globalVendorListLocation: 'https://vendorlist.consensu.org/vendorlist.json',
+                      // customPurposeListLocation: './purposes.json',
+                      // globalConsentLocation: './portal.html',
+                      // storeConsentGlobally: false,
+                      // storePublisherData: false,
+                      logging: 'debug',
+                      // localization: {},
+                      // forceLocale: 'en-us'
+                  }
+                  return cmp;
+              }());
+              var t = document.createElement('script');
+              t.async = false;
+              t.src = 'http://acdn.adnxs.com/cmp/cmp.bundle.js';
+              var tag = document.getElementsByTagName('head')[0];
+              tag.appendChild(t);
+          }
+      })(window, document);
+      window.__cmp('showConsentTool');
+  </script>
+
+      <script>
+        var PREBID_TIMEOUT = 700;
+        var FAILSAFE_TIMEOUT = 2500;
+
+        var adUnits = [{
+          code: 'div-gpt-ad-1460505748561-0',
+          mediaTypes: {
+            banner: {
+              sizes: [[300, 250], [300,600]]
+            }
+          },
+
+          // Replace this object to test a new Adapter!
+          bids: [{
+            bidder: 'appnexus',
+            params: {
+              placementId: 13144370
+            }
+          }]
+
+        }];
+
+        var pbjs = pbjs || {};
+        pbjs.que = pbjs.que || [];
+
+      </script>
+
+      <script>
+        var googletag = googletag || {};
+        googletag.cmd = googletag.cmd || [];
+        googletag.cmd.push(function() {
+          googletag.pubads().disableInitialLoad();
+        });
+
+        _tcfapi('addEventListener', 2, function(tcData, success) {
+          if (tcData.eventStatus === 'useractioncomplete') {
+              var slotCodes = tagSlots
+                  .map( function(slot) { return slot.getSlotElementId() || ''; } )
+                  .filter( function(id) { return id; } );
```

Comment:
Do we need this step? We could likely just do a pbjs.addAdUnits(adUnits) before registering this event listener.

Reply:
No we dont. Yeah. Got rid of this. Thanks :)